spec/index.bs

```diff
@@ -137,7 +137,7 @@ In order to reduce the risk, the WebGPU working group will coordinate with GPU v
 to integrate the WebGPU Conformance Test Suite (CTS) as part of their driver testing process,
 like it was done for WebGL.
 WebGPU implementations are expected to have workarounds for some of the discovered bugs,
-and support blacklisting particular drivers from using some of the native API backends.
+and support blocklisting particular drivers from using some of the native API backends.
```

Comment:
This sentence didn't really quite make sense to begin with, how about let's just edit it now.
```suggestion
and disable WebGPU on drivers with known bugs that can't be worked around.
```